apps/synapse-oauth-signin/src/OAuth2Form.tsx

```diff
@@ -319,6 +330,9 @@ export function OAuth2Form() {
 
   return (
     <StyledOuterContainer>
+      <Backdrop open={!hasInitializedSession} sx={{ zIndex: 5 }}>
```

Comment:
zIndex required because the MUI buttons were above the backdrop (everything else was below)